topologic/embedding/node2vec_embedding.py

```diff
@@ -100,12 +103,262 @@ def _learn_embeddings(walks: list,
     walks = [list(map(str, walk)) for walk in walks]
 
     # Documentation - https://radimrehurek.com/gensim/models/word2vec.html
-    model = Word2Vec(walks,
-                     size=dimensions,
-                     window=window_size,
-                     min_count=0,
-                     sg=1,  # Training algorithm: 1 for skip-gram; otherwise CBOW
-                     workers=workers,
-                     iter=iterations)
+    model = Word2Vec(
+        walks,
+        size=dimensions,
+        window=window_size,
+        min_count=0,
+        sg=1,  # Training algorithm: 1 for skip-gram; otherwise CBOW
+        workers=workers,
+        iter=iterations
+    )
 
     return model
+
+
+class _Node2VecGraph:
```

Comment:
In general I feel like this class is in dire need of some actually descriptive variable names.  G, J, K, kk, p, q - this might just be me as a software developer looking at this but it just screams "you'll never figure out what this means unless you have read 1..n whitepapers first".  Can we do any better?

Reply:
There are a couple links for additional reading in the code. I renamed the variables to the best of my ability.